data/templates/scripts/to_ducky_script.template

```diff
@@ -0,0 +1,15 @@
+REM Title: Metasploit Generated Payload
```

Comment:
If the other comment is taken, this file should also be renamed so we can add more templates in the future that use different techniques and/or support more platforms.

`to_powershell.ducky_script.template` looks like it'd follow the convention set by `to_powershell.hta.template` and `to_powershell.vba.template`.